### ISSUE TITLE
Add Recommendation to Screenshot Display Settings

### ISSUE THREAD

**Issue by FlorianWendelborn** [tonsky/tonsky.github.io#70]
For some monitors, the supported set of features around scaling is different. I think it’d be great to add this as an optional(?) screenshot to newly created issues.

E.g.

![image](https://user-images.githubusercontent.com/1133858/163898820-f657e118-e99d-4e2c-b4f1-f467c25b0af5.png)

Vs

![image](https://user-images.githubusercontent.com/1133858/163898919-c1b5729f-fcbe-4e97-8e56-c1d21aab43eb.png)
(source: <https://www.eizoglobal.com/support/compatibility/dpi_scaling_settings_mac_os_x/>

**Comment by tonsky** [comment]
You mean for https://tonsky.me/blog/monitors-mac/? Screenshots there are just to confirm hardware specs

**Comment by FlorianWendelborn** [comment]
Yes, this would confirm that the monitors aren't just supported but well supported by macOS for scaling. I've had a couple cases with screens that don't or do support this and it's pretty hard to find out right now. So this list could help a lot with this

**Comment by tonsky** [comment]
You can see it better on a screenshot like this

![image](https://user-images.githubusercontent.com/285292/164003936-3d05506a-62ce-44fa-96aa-e2567aff9f14.png)

**Comment by FlorianWendelborn** [comment]
@tonsky can you elaborate? I can’t see the scaling support on that screenshot.

**Comment by tonsky** [comment]
<img width="281" alt="Screenshot 2022-04-19 at 15 12 13" src="https://user-images.githubusercontent.com/285292/164020787-26f8ac4a-67d5-495c-ba3f-2171311c1ec5.png">

Here’s the native monitor resolution. What are you looking for in the scaling menu? List of options monitor can scale to? Are you interested in one particular scale?

**Comment by FlorianWendelborn** [comment]
Yes, I’m talking about the available scaling options as they vary by hardware support in (afaik) pretty unpredictable patterns.

**Comment by FlorianWendelborn** [comment]
Basically, display/GPU combinations that support scaling are at least IMO objectively better for macOS as you can scale them without sacrificing resolution. If you were to attempt to try to scale my display, you’d have to switch it to 720p which is stupid
